src/OptimTestProblems.jl

```diff
@@ -1,8 +1,11 @@
 module OptimTestProblems
 
+import LinearAlgebra: gradient
```

Comment:
Another "unnecessary" gradient ;)

Reply:
didn't you have a thesis to submit???

Reply:
Haha, yes I'm gonna leave it to you now :laughing:

Reply:
congrats btw 🎆 (is it submitted?)

Reply:
I need to temporarily turn off GitHub emails :p

Reply:
In a few hours; UK academia is still fast asleep :see_no_evil: